graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/DXGraphQLProvider.java

```diff
@@ -283,6 +283,14 @@ public void activate() {
         return subscriptionType.getFieldDefinitions();
     }
 
+    @Override
+    public Collection<GraphQLFieldDefinition> getFields() {
+        /*
+         * FIXME: part of interface for both mutation and subscription
+         */
+        return null;
```

Comment:
@tdraier this one not sure how to resolve. the mutation and subscription provider interfaces has the same method. Is there a way to tell if it's a mutation or subscription provider? Or maybe we should rename `getMutationFields` and `getSubscriptionFields` ?

Reply:
getFields is used in servlet, for both mutation and subscription - the only way I see is to split the providers for these 2 interfaces